src/Model/APContact.php

```diff
@@ -350,26 +352,61 @@ public static function getByURL($url, $update = null)
 		return $apcontact;
 	}
 
+	/**
+	 * Mark the given AP Contact as "to archive"
+	 *
+	 * @param array $apcontact
+	 * @return void
+	 */
+	public static function markForArchival(array $apcontact)
+	{
+		if (!empty($apcontact['inbox'])) {
+			Logger::info('Set inbox status to failure', ['inbox' => $apcontact['inbox']]);
+			HTTPSignature::setInboxStatus($apcontact['inbox'], false);
+		}
+
+		if (!empty($apcontact['sharedinbox'])) {
+			// Check if there are any vital inboxes
```

Comment:
"vital"? Did you mean "active" or "available"?

Reply:
I mean not failed ones.

Reply:
Please change to "available" then. Vital means "of a crucial importance" or "necessary to sustain life".

Reply:
Yes, Mr. English teacher ;-)

Reply:
Well, someone has to do it!